create_catalog.py

```diff
@@ -19,8 +19,15 @@
 
 
 def scrub_unicode(text):
-    return unicodedata.normalize('NFKD', unicode(text)).encode('ascii', 'ignore')
-
+    '''
+    Process special characters used on the HTML encoding to
+    ASCII or UTF-8
+    '''
+    try:
+        return unicodedata.normalize('NFKD', unicode(text)).encode('ascii', 'ignore')
+    except NameError as ne:
+        print(ne)
+        return str(text)
```

Comment:
This satisfies the Python 3 unicode requirement.

Reply:
Nice!

Reply:
Actually, this is outdated... 
I've got a much shorter fix for this. Not sure why it hasn't made it into the PR...